src/containers/pages/OrganizationViews/helpers/tests/servicehooks.test.ts

```diff
@@ -17,7 +17,9 @@ describe('src/containers/pages/OrganizationViews/helpers/servicehooks', () => {
       };
     });
 
-    loadOrganization('organizationId', mockClass, mockActionCreator);
+    loadOrganization('organizationId', mockClass, mockActionCreator).catch(e => {
+      throw e;
+    });
```

Comment:
If we don't want to add mock growl alerts in these tests then it feels like we should at least throw the error like this.